src/wms/src/main/java/org/geoserver/wms/map/DirectRasterRenderer.java

```diff
@@ -95,6 +95,10 @@ class DirectRasterRenderer {
 
     private static final int MAX_TILE_SIZE = 1024;
 
+    private static final String MAP_WRAPPING_FORMAT_OPTION = "mapWrapping";
+    private static final String ADV_PROJECTION_HANDLING_FORMAT_OPTION =
```

Comment:
Rather than duplicating constants and extraction method (getFormatOptionsAsBoolean) make them static public and use them directly.

Reply:
Fixed